### ISSUE TITLE
Question: BlockEditor vs gutenberg-js

### ISSUE THREAD

**Issue by ashwinputhige** [front/gutenberg-js#34]
[BlockEditor](https://developer.wordpress.org/block-editor/packages/packages-block-editor/) in gutenberg says that `This module allows you to create and use standalone block editors.`  . Is this library the same as using the `@wordpress/block-editor` npm package from gutenberg?

**Comment by SofiaSousa** [comment]
Yes, it's the same. Gutenberg-js was just a way to put all the Gutenberg packages together when they weren't published as npm public packages. 

Note that Gutenberg-js is **deprecated**.

**Comment by PedroPotango** [comment]
I am also a bit deprecated. Thanks for the great work!
